src/zzz_od/api/routers/battle_assistant.py

```
@@ -0,0 +1,2472 @@
+
+    return {
+        "is_in_battle": is_in_battle,
+        "current_action": current_action,
+        "enemies_detected": enemies_detected,
+        "last_update": datetime.now().isoformat(),
+        "performance_metrics": performance_metrics
+    }
+
+
+@router.get("/detailed-battle-state", response_model=DetailedBattleState, summary="获取详细战斗状态")
+def get_detailed_battle_state(ctx: ZContext = Depends(get_ctx)):
```

Ruff (0.12.2)

Redefinition of unused `get_detailed_battle_state` from line 1931

(F811)